.github/workflows/release.yml

```
@@ -136,6 +136,25 @@ jobs:
             - name: Run zepter
               run: zepter run check
 
+    shellcheck:
+        runs-on: ubuntu-latest
+        needs: ["set-tags"]
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v6
+            with:
+                ref: ${{ needs.set-tags.outputs.git_ref }}
+          - name: Run Shellcheck for test scripts
+            uses: ludeeus/action-shellcheck@v2
+            with:
+              severity: error
+              scandir: './test/scripts'
+          - name: Run Shellcheck for tools
+            uses: ludeeus/action-shellcheck@v2
```

My bad, try this one:

```suggestion
            uses: ludeeus/action-shellcheck@2.0.0
```